ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -332,6 +344,17 @@ Loader {
             }
         }
 
+        function convertOutgoingStatus(value) {
```

Comment:
I wonder if we really those now that StatusQ lives in the same codebase. I think we can use `Contstants` for everything now?

Reply:
As good guys we still should keep StatusQ an independent library.
And actually we have PR checks that will not allow me to do that 😄 

Donno. That's a very big and very uninmportant question right now 😄